### ISSUE TITLE
How to save prediction label as binary mask one by one?

### ISSUE THREAD

**Issue by Hongxing-Cui** [Media-Smart/vedaseg#26]
How to save prediction label as binary image on by one, which name is same to the corresponding of image name from every patch images?

**Comment by DarthThomas** [comment]
Hi, 
Thanks for your interest in our VedaSeg, we are currently working on the inference part, you'll be able to modify and save predictions as you wish after code release.

**Comment by Hongxing-Cui** [comment]
Hi, How to visualize the net architecture of configs based on the python code, thanks. @DarthThomas

**Comment by mileistone** [comment]
> How to save prediction label as binary image on by one, which name is same to the corresponding of image name from every patch images?

We support this feature now.

**Comment by DarthThomas** [comment]
> Hi, How to visualize the net architecture of configs based on the python code, thanks. @DarthThomas

There are many different ways to visualize your model. Since we are using PyTorch, probably one of the easiest ways is to just `print(model)`. You can refer to [this link](https://discuss.pytorch.org/t/is-there-similar-pytorch-function-as-model-summary-as-keras/2678)
